### PR TITLE
Fix display of failed txs in wallets

### LINKED ISSUE #138: [EPIC] Port explorer logic & display to the wallets to better display failed transaction
When a transaction fails, it is currently not clearly displayed in the mobile & desktop wallets (amounts are white VS successful transactions are colored). However, it is very cleary/explicitly shown in the explorer. 

**Describe the solution you'd like**
Port explorer logic & display to the wallets to better display failed transaction

## Subtasks
- [ ] #335
- [ ] #336

### LINKED ISSUE #335: Improve desktop wallet display of failed txs by porting code from explorer

### LINKED ISSUE #336: Improve mobile wallet display of failed txs by porting code from explorer

### PR DESCRIPTION
**Pull request by nop33**
Closes #138
Closes #335
Closes #336

This has not been tested, I need to find a testnet wallet that has some failed transactions. If you know of any, please let me know.

This is how it would look like

<details><summary>See screenshots</summary>

### Tx list
image

### Tx details
image

### Tx list
image

### Tx details
image

</details>

### CODE REVIEW

File: apps/desktop-wallet/src/components/TransactionalInfo.tsx [hunk 1]
```diff
@@ -84,6 +87,9 @@ const TransactionalInfo = ({
         <CellArrow>
           <TransactionIcon color={iconBgColor}>
             <Icon size={13} strokeWidth={3} color={iconColor} />
+            <FailedTXBubble data-tooltip-id="default" data-tooltip-content={t('Script execution failed')}>
```
Comment (mvaivre) [inline apps/desktop-wallet/src/components/TransactionalInfo.tsx:90]: I think we need to add a condition here :)

Comment (nop33) [reply apps/desktop-wallet/src/components/TransactionalInfo.tsx:90]: Ah right, thanks for catching it, I was testing how it looks :D

Comment (nop33) [reply apps/desktop-wallet/src/components/TransactionalInfo.tsx]: Done!

**Review by mvaivre — APPROVED** [review]
👍

**Pull request merged**